.github/workflows/tag.yml

```diff
@@ -0,0 +1,24 @@
+name: tag
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  tag:
+    name: Tag capa rules
+    runs-on: ubuntu-20.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
The reason why I use ubuntu-20.04, is because I read this in the GitHub documentation:
> Note: The ubuntu-latest label is currently being migrated from the Ubuntu 18.04 virtual environment to Ubuntu 20.04. Specifying the ubuntu-latest label during the migration might use either virtual environment. To explicitly use Ubuntu 20.04 or Ubuntu 18.04, specify ubuntu-20.04 or ubuntu-18.04. When the migration is complete, ubuntu-latest will use the Ubuntu 20.04 virtual environment. For more information about the migration, see "Ubuntu-latest workflows will use Ubuntu-20.04."

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

As this action is simple and doesn't use anything that should be version specific, we could use `ubuntu-latest`, but in general I find not knowing which ubuntu version is every time used a bit dangerous (which happens every time it is migrated). So I though that sticking which one we know for sure that works may be better. But I think either way is ok. Do you want that I change it?

Reply:
Ui, good catch. We should better pin a concrete version for all tests then?!
For example, Ubuntu 20 doesn't come with Python 2, although I think we will dodge that bullet by going Py3 only soon 🤞

Reply:
#490